src/components/AsideHeader/AsideHeader.module.scss

```
@@ -194,6 +233,35 @@ $block: '.#{variables.$ns}aside-header';
 
+    &__quick-access {
+        position: relative;
+        flex: none;
+        box-sizing: border-box;
+        width: 100%;
+        padding-block: var(--gn-aside-header-section-divider-gap);
+        margin-bottom: var(--gn-aside-header-section-divider-gap);
+
+        &::after {
+            @include section-divider-line;
+            bottom: 0;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)